dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue

```diff
@@ -174,8 +174,8 @@
           formData.append('file', this.file)
           formData.append('type', this.type)
           formData.append('name', this.name)
-          formData.append('pid', this.pid)
-          formData.append('currentDir', this.currentDir)
+          formData.append('pid', localStore.getItem('pid') || this.id)
+          formData.append('currentDir', localStore.getItem('currentDir') || this.currentDir)
```

Comment:
This problem may also appear in `definitionUpdate.vue`, `fileChildReUpdate.vue`, `fileReUpload.vue`, `fileUpdate.vue`, `resourceChildUpdate.vue` and `udfUpdate.vue` under directory `dolphinscheduler-ui/src/js/module/components/fileUpdate`, please check together, thanks a lot!

Reply:
ok,I will check it

Reply:
I fix all bugs for upload resource file and udf,by the way,I fix when delete udf resource file,the page not refresh